### DISCUSSION TITLE
Discharge battery manually

### DISCUSSION THREAD

**Discussion by Batterysolarguru** [amberelectric/public-api#147] [Ideas]
Can I use the amber platform and control how much I feed in and when  I want to is that possible as I can control my inverter with battery to feed in Max 5kwh battery power l am with amber at the moment and wonder if I was to feed in with out tesla or sungrow
LG will I still get paid 
Thanks
Battery Solar Guru

**Comment by madpilot** [comment]
If you can control your battery, and discharge power to the grid, you will get paid for those exports.

**Reply by Batterysolarguru** [reply]
How can I monitor it through amber as my acc at the moment has no feed in

**Reply by madpilot** [reply]
Email info@amber.com.au - they can give you the info you need to get a feed in channel

**Reply by Batterysolarguru** [reply]
OK thanks I am just getting my head around the basics before I buy amber battery ready system like sungrow to be automated
New at git hub as well so having trouble finding answers 
Thanks again
